src/components/ClientSideLoggingToolMenu/index.android.tsx

```diff
@@ -1,47 +1,5 @@
-import React, {useState} from 'react';
-import RNFetchBlob from 'react-native-blob-util';
-import Share from 'react-native-share';
-import type {Log} from '@libs/Console';
-import localFileCreate from '@libs/localFileCreate';
-import CONST from '@src/CONST';
-import BaseClientSideLoggingToolMenu from './BaseClientSideLoggingToolMenu';
-
 function ClientSideLoggingToolMenu() {
-    const [file, setFile] = useState<{path: string; newFileName: string; size: number}>();
-
-    const createAndSaveFile = (logs: Log[]) => {
-        localFileCreate('logs', JSON.stringify(logs, null, 2)).then((localFile) => {
-            RNFetchBlob.MediaCollection.copyToMediaStore(
-                {
-                    name: localFile.newFileName,
-                    parentFolder: '',
-                    mimeType: 'text/plain',
-                },
-                'Download',
-                localFile.path,
-            );
-            setFile(localFile);
-        });
-    };
-
-    const shareLogs = () => {
-        if (!file) {
-            return;
-        }
-        Share.open({
-            url: `file://${file.path}`,
-        });
-    };
-
-    return (
-        <BaseClientSideLoggingToolMenu
-            file={file}
-            onEnableLogging={() => setFile(undefined)}
-            onDisableLogging={createAndSaveFile}
-            onShareLogs={shareLogs}
-            displayPath={`${CONST.DOWNLOADS_PATH}/${file?.newFileName ?? ''}`}
-        />
-    );
+    return null;
```

Comment:
@dominictb Instead of removing all the code altogether for ClientSideLoggingToolMenu Android and iOS files, can we please hide it on the platforms? I am not sure if we will need this code in the near future. What are your thoughts on removing this code @Julesssss ?

Reply:
> Instead of removing all the code altogether for ClientSideLoggingToolMenu Android and iOS files, can we please hide it on the platforms? I am not sure if we will need this code in the near future.

I'm slightly confused by the need to keep this. Is it just in case we change the way we trigger the debug modal?

Reply:
> [Only client side logging is currently supported on web and desktop natively right now, so the menu options should reflect that.
](https://github.com/Expensify/App/issues/43256#issuecomment-2154610196)

Right now, client-side logging is available for the web and desktop, so we need to reflect that in the menu options. I was thinking about when we enabling it for Android and iOS in the future we need to write this code again

Reply:
> Perhaps instead we could leave a comment and link to this PR

Thanks for the suggestion. I agree with this; let's add a comment to link this PR @dominictb

Reply:
Remove all code from both `ClientSideLoggingToolMenu/index.ios.tsx` and `ClientSideLoggingToolMenu/index.android.tsx` should return null(right now its correct). Additionally, add a comment in both files indicating that the code was removed in this PR, including the link to the PR and the reason for its removal.